birdhouse/optional-components/stac-populator/docker-compose-extra.yml

```diff
@@ -0,0 +1,21 @@
+version: "3.4"
+
+x-logging:
+  &default-logging
+  driver: "json-file"
+  options:
+    max-size: "50m"
+    max-file: "10"
+
+services:
```

Comment:
Correct me if I'm wrong but doesn't this automatically populate the stac catalog with cmip5 and cmip6 by default: 

https://github.com/crim-ca/stac-populator/blob/master/populate.sh#L40-L43

That seems reasonable to demonstrate the catalog but not very useful for actual usage. We should be able to configure what entries are in the catalog (raised in https://github.com/crim-ca/stac-populator/issues/2).

Reply:
Also, right now, this component will run once when we call `./pavics-compose.sh up` and then exit when its done. That goes against the idea of a "service" in compose, which should really be a long running service, not a single-run process. 

It's also not that intuitive how to use this to re-populate the stac instance once the birdhouse stack is already up and running.

I think that this makes more sense as a script  in (`birdhouse/scripts`) that calls:

`docker run --rm --network container:stac --name stac-populator -e STAC_ASSET_GENERATOR_TIMEOUT=30 -e STAC_HOST=http://stac:8000/stac ghcr.io/crim-ca/stac-populator:master  ./populate.sh`